plugins/modules/database_view_info.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: database_view_info
+
+author:
+  - Toni Moreno (@toni-moreno)
+
+short_description: a module that users can use to query on any previously defined database_view
+
+description:
+  - query on any previously defined database_view
+version_added: 1.6.0
+extends_documentation_fragment:
+  - servicenow.itsm.instance
+  - servicenow.itsm.query
+options:
+  state:
```

Comment:
`type` and `description` are mandatory fields.

Reply:
ok fixed now